pyproject.toml

```diff
@@ -27,7 +27,13 @@ classifiers = [
 include = ["MANIFEST.in"]
 
 [tool.poetry.dependencies]
-python = "^3.6"
+python = "^3.8"
```

Comment:
In order to use last version of `pytest` we need to update the minimum python version. Also, 3.6 is [absolutely dead](https://devguide.python.org/versions/):
![image](https://github.com/user-attachments/assets/7cf6187d-fbd6-4a7d-b58f-d3aadec1e016)